From 68ab639c5164805d295c0173f918116dd2a6d199 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Wed, 25 Mar 2026 20:05:24 +0800
Subject: [PATCH] configurations: nvidia: clover_cx9: add NIC sensor configs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add NIC sensor configuration to enable sensor monitoring.

[1] Gerrit: https://gerrit.openbmc.org/c/openbmc/entity-manager/+/88679

Tested:
```
root@bmc:~# busctl tree xyz.openbmc_project.EntityManager | grep -e "NS_NIC"
          │ ├─ /xyz/openbmc_project/inventory/system/board/NVIDIA_VR_NVL72_Clover/NS_NIC_PWR_W
          │ ├─ /xyz/openbmc_project/inventory/system/board/NVIDIA_VR_NVL72_Clover/NS_NIC_TEMP_C
```

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
Change-Id: I3fc20fb3a25483d6aaaa72a4da41d345d15bd0c4
---
 configurations/nvidia/clover_cx9.json | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/configurations/nvidia/clover_cx9.json b/configurations/nvidia/clover_cx9.json
index 1129750..5abc559 100644
--- a/configurations/nvidia/clover_cx9.json
+++ b/configurations/nvidia/clover_cx9.json
@@ -5,6 +5,24 @@
             "Bus": "$bus",
             "Name": "NVIDIA VR NVL72 Clover FRU",
             "Type": "EEPROM"
+        },
+        {
+            "MaxValue": 255,
+            "MinValue": -255.0,
+            "Name": "NS_NIC_TEMP_C",
+            "PowerState": "On",
+            "Timeout": 30.0,
+            "Type": "ExternalSensor",
+            "Units": "DegreesC"
+        },
+        {
+            "MaxValue": 65535,
+            "MinValue": 0,
+            "Name": "NS_NIC_PWR_W",
+            "PowerState": "On",
+            "Timeout": 30.0,
+            "Type": "ExternalSensor",
+            "Units": "Watts"
         }
     ],
     "Name": "NVIDIA VR NVL72 Clover",
-- 
2.53.0-Meta

